src/io/hdf/tests/test_hdf_archive.cpp

```diff
@@ -14,6 +14,7 @@
 #include "catch.hpp"
 
 #include "hdf/hdf_archive.h"
+#include <H5Fpublic.h>
```

Comment:
We tried to prevent hdf5 library details from the users of hdf_archive. Is this include really needed?

Reply:
Seemed removable, I'm not sure why I didn't remove it with whatever was getting used from it in the test program.